src/transport/http/binary_emitter.ts

```diff
@@ -0,0 +1,25 @@
+import axios, { AxiosRequestConfig } from "axios";
+
+import { CloudEvent, Version } from "../../event";
+import { Options } from "../emitter";
+import { Headers, headersFor } from "./headers";
+import { asData } from "../../event/validation";
+
+export async function emitBinary(event: CloudEvent, options: Options): Promise<object> {
+  if (event.specversion !== Version.V1 && event.specversion !== Version.V03) {
+    return Promise.reject(`Unknown spec version ${event.specversion}`)
+  } else {
+    return emit(event, options, headersFor(event));
+  }
+}
+
+async function emit(event: CloudEvent, options: Options, headers: Headers): Promise<object> {
+  const contentType: Headers = { "content-type": "application/json; charset=utf-8" };
+  const config = {
+    ...options,
+    method: "POST",
+    headers: { ...contentType, ...headers },
```

Comment:
I sent this PR to your branch, https://github.com/lance/sdk-javascript/pull/1 

If a custom header is passed in the options like 
```
{ headers: { customHeader: "value" } }
```
It was actually getting overwritten since the spread operator does a shallow merge(i think)

Reply:
Thanks! I've merged that change. :)